CONTRIBUTING.md

```diff
@@ -24,42 +24,55 @@ GitHub:
     want to read [Working with Submodules](https://github.blog/2016-02-01-working-with-submodules/)
     for a better understanding of how this works.
 
-3. Remove any previously installed librosas:
-            
-          $ pip uninstall librosa
+3. Set the upstream remote to the Librosa's repo:
 
-and install your local copy with testing dependencies:
+          $ git remote add upstream git@github.com:librosa/librosa.git 
 
-          $ pip install -e .[tests]
+4. Create a new conda environment in order to install dependencies:
 
-4. Create a branch to hold your changes:
+          $ conda create -n librosa-dev python=3.9
 
-          $ git checkout -b my-feature
+          $ conda env update -n librosa-dev --file .github/environment-ci.yml
+
+          $ conda activate librosa-dev
+
+          $ pip install -e '.[tests]'
+
+5. Create a branch to hold your changes:
+
+          $ git checkout -b <NAME-NEW-BRANCH>
```

Comment:
Would it be a bad idea to use the (newer) simpler syntax
```
$ git switch -c <NAME-NEW-BRANCH>
```
?

I generally prefer it (since checkout is confusing) but it does require/assume a fairly recent version of git (2.23+).

Reply:
We should totally use the newer syntax so people get familiar with it!
```suggestion
          $ git switch -c <NAME-NEW-BRANCH>
```